tests/markdown.js

```diff
@@ -10,10 +11,12 @@ define([
     'text!static/markdown/simple-markdown-no-chars.xml',
     'text!static/markdown/no-markdown.xml',
     'text!static/markdown/no-markdown-stars.xml',
-    'text!static/markdown/explicit-no-markdown.xml'
+    'text!static/markdown/explicit-no-markdown.xml',
+    'text!static/markdown/markdown-output-value.xml'
 ], function (
     options,
     util,
+    vellum_util,
```

Comment:
Looks like we use both conventions. tests/widgets uses vellumUtil but databrowser and javaRosa use vellum_util. Should `vellumUtil` be the canonical form?